<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityHouse</class>
 <widget class="QMainWindow" name="ActivityHouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="labelAddress">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Melyik ház?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxBills">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(221, 234, 240);
</string>
      </property>
      <property name="title">
       <string>Rezsi költségek</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelDate">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dátum:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="locale">
            <locale language="Hungarian" country="Hungary"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="displayFormat">
            <string>yyyy/MMMM</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelWaterBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Víz és csatorna:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxWaterBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelGasBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Gáz:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxGasBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelElectricityBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Villany:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxElectricityBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelInternetBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Internet - TV:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxInternetBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelCommonBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Közös költség:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CurrencySpinBox" name="spinBoxCommonBill">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(173, 216, 230);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QToolButton" name="toolButtonInsurance">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 216, 230);</string>
        </property>
        <property name="text">
         <string>Biztosítás</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameInsurance">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(221, 234, 240);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="labelInsurance">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Biztosítási díj:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="CurrencySpinBox" name="spinBoxInsurance">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 216, 230);</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelInsuranceYear">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Biztosítási év:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEditInsurance">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(173, 216, 230);</string>
             </property>
             <property name="displayFormat">
              <string>yyyy</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QToolButton" name="toolButtonOther">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 216, 230);</string>
        </property>
        <property name="text">
         <string>Egyéb költség</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameOther">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(221, 234, 240);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="labelOtherCost">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Egyéb költség:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="CurrencySpinBox" name="spinBoxOtherCost">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(173, 216, 230);</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelOtherDate">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Dátum:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditOtherCost">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(173, 216, 230);</string>
               </property>
               <property name="locale">
                <locale language="Hungarian" country="Hungary"/>
               </property>
               <property name="displayFormat">
                <string>yyyy.MMMM.dd</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="labelComment">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Leírás:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditComment">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(173, 216, 230);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pushButtonSave">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(173, 216, 230);
font: 700 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Mentés</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurrencySpinBox</class>
   <extends>QSpinBox</extends>
   <header>c:/Users/Dani/Qt-source/FamilyFinancies/FamilyFinancies/currencyspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
